src/runtime/crdt/tests/crdt-count-test.ts

```diff
@@ -77,40 +84,55 @@ describe('CRDTCount', () => {
 
     assert.isTrue(otherChange.changeIsOperations);
     assert.equal(otherChange.operations.length, 1);
-    assert.deepEqual(otherChange.operations[0], {actor: 'me', value: 3, type: CountOpTypes.MultiIncrement});
+    assert.deepEqual(otherChange.operations[0], {actor: 'me', value: 3, type: CountOpTypes.MultiIncrement, version: {from: 1, to: 2}});
 
     assert.isTrue(count2.applyOperation(otherChange.operations[0]));
     assert.deepEqual(count1.getData(), count2.getData());
   });
 
+  it('throws when attempting to merge divergent models', () => {
+    const count1 = new CRDTCount();
+    const count2 = new CRDTCount();
+    count1.applyOperation({type: CountOpTypes.MultiIncrement, actor: 'me', value: 7, version: {from: 0, to: 1}});
+    count2.applyOperation({type: CountOpTypes.MultiIncrement, actor: 'me', value: 4, version: {from: 0, to: 1}});
+    assert.throws(() => count1.merge(count2));
```

Comment:
May as well check the error type: CRDTError

Reply:
done.